modules/openapi-generator/src/main/resources/python/python-experimental/api.mustache

```diff
@@ -105,7 +105,7 @@ class {{classname}}(object):
                     Default is True.
                 _host_index (int): specifies the index of the server
                     that we want to use.
-                    Default is 0.
+                    Default is read from the configuration.
```

Comment:
Can you update the docstring here to show that _host_index can now be type `int/None`?